CRM/Core/BAO/Log.php

```diff
@@ -42,18 +42,22 @@ class CRM_Core_BAO_Log extends CRM_Core_DAO_Log {
    * @param string $table
    *
    * @return array|null
+   *
+   * This just has a couple of small changes suggested by Eileen to make this function fail more gracefully.
```

Comment:
This comment doesn't have much meaning once this change is committed and should probably be removed.  It's more of a "commit message" comment